### DISCUSSION TITLE
Add ID3 [batch] editing and another multi-rename tool

### DISCUSSION THREAD

**Discussion by anonimo82** [doublecmd/doublecmd#1824] [Suggestions]
Hello everyone.
As in title, please, add the feature to modify ID3 tags for Mp3s and the like, then maybe rename files accordingly in batch.
Something you'd do with Mp3tag (mp3tag.de/en), but with DoubleCMD, which I like more.

Thanks in advance for any help.
Regards, have a nice day.

**Comment by JonnyTech** [comment]
https://github.com/j2969719/doublecmd-plugins/tree/master/plugins/wfx/taglib_crap

**Comment by anonimo82** [comment]
I'm having a bad time compiling that, please, can you help me?
Edit: Nevermind, I'll ask for help in the repo

**Comment by anonimo82** [comment]
Uhm, it's very basic functionality.
It should edit tags in batch, and rename in batch.

**Reply by j2969719** [reply]
yep thats just some lazy shitcode + wfx api has some limitations
atm, to fill tags/convert several files, you need to select files and press f5 there..

**Comment by alexx2000** [comment]
You can rename files according tags using [multi-rename](https://doublecmd.github.io/doc/en/multirename.html) tool and standard AudioInfo plugin.

**Comment by anonimo82** [comment]
I see, thanks a lot.
I guess the multi rename based on tags could be simplified, but this already works very well.
Thanks a lot!!!

Regards.

**Comment by anonimo82** [comment]
Hello again. Any other plugin working on Windows?

Edit: MediaFactory
